.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
+.idea/* linguist-generated=false
```

Comment:
Is this for github's sidebar showing languages for current repo?

<img width="330" alt="image" src="https://user-images.githubusercontent.com/1264761/219879327-d63863ad-2749-4c77-9f08-b1312c4aac7e.png">

Reply:
Not quite.

- `* text=auto` makes sure that `autocrlf` is enabled, even if the config option isn't set on a devs local machine.
- `linguist-generated=false` makes it so that the diff is not hidden in a PR for those files.